src/components/Api.js

```diff
@@ -1,30 +1,26 @@
 export class Api {
   constructor(config) {
-    this.url = config.baseUrl;
-    this.headers = config.headers;
+    this._url = config.baseUrl;
+    this._headers = config.headers;
   }
 
+  _processResponse = (res) => {
+    if (res.ok) {
+      return res.json();
+    }
+    return Promise.reject(`Error: ${res.status}`);
+  };
+
   getInitialCards() {
-    return fetch(`${this.url}/cards`, {
-      headers: {
-        authorization: "fe7e07a4-81c5-490b-807b-e6a7cec619a0",
-      },
-    }).then((res) => {
-      if (res.ok) {
-        return res.json();
-      }
-      // if the server returns an error, reject the promise
-      return Promise.reject(`Error: ${res.status}`);
-    });
+    return fetch(`${this._url}/cards`, {
+      headers: this._headers,
+    }).then(this._processResponse);
   }
 
   getUserInfo() {
-    return fetch("https://around-api.en.tripleten-services.com/v1/users/me", {
+    return fetch(`${this._url}/users/me`, {
       method: "GET",
-      headers: {
-        authorization: "fe7e07a4-81c5-490b-807b-e6a7cec619a0",
-        "Content-Type": "application/json",
-      },
+      headers: this._headers,
```

Comment:
Needs correction: use `this._processResponse` to process all the api responses (there should not be code duplication in all the api methods)

Reply:
I changed the "res" in ".then" to "this._processResponse."  I saw "headers: this._headers" is highlighted; should I have replaced that with "this._processResponse"? Or have I updated the code correctly? Everything seems to be working as it should. Thank you.